scanomatic/ui_server_data/js/ccc/components/PolynomialResultsColonyHistogram.jsx

```diff
@@ -106,10 +106,18 @@ export default class PolynomialResultsColonyHistogram extends React.Component {
           ref={this.refBinder}
           id={`poly-colony-chart-${colonyIdx}`}
         />
-        <span className="poly-colony-txt">{valueFormatter(independentMeasurement, 2)} cells</span>
+        <span className="poly-colony-txt">{valueFormatter(independentMeasurements, 2)} cells</span>
       </div>
     );
   }
 }
 
-PolynomialResultsColonyHistogram.propTypes = CCCPropTypes.colonyShape;
+PolynomialResultsColonyHistogram.propTypes = {
```

Comment:
was it only used here or why this change?

Reply:
This is a reversal of a change from a previous PR, which didn't quite work (gave the weird error ```ERROR: 'Warning: Failed prop type: The prop `isRequired` is marked as required in `PolynomialResultsColonyHistogram`, but its value is `undefined`.
    in PolynomialResultsColonyHistogram'``` even though the linter didn't complain). The `colonyShape` was only used here anyway.

Reply:
then it must be removed yes.

Reply:
done